src/wallet/walletdb.h

```diff
@@ -212,6 +212,20 @@ class CWalletDB : public CDB
         nWalletDBUpdated++;
         return Write(std::string("backuptime"), backup_time);
     }
+
+    template<typename T>
+    bool WriteAttribute(const std::string& attribute, const T& value)
+    {
+        nWalletDBUpdated++;
+        return Write(std::make_pair(std::string("attribute"), attribute), value);
+    }
+
+    template<typename T>
+    bool ReadAttribute(const std::string& attribute, T& value)
+    {
+        nWalletDBUpdated++;
```

Comment:
```suggestion
```
Why's the update counter increased on a read?

Reply:
Good catch. Agreed.